.github/workflows/tests.yaml

```diff
@@ -18,7 +18,7 @@ jobs:
       - name: run the tests
         run: python3 tests.py 
       - name: verify type hints
-        run: mypy datemath
+        run: mypy . --strict
```

Comment:
Running on tests is important too. This ensures expected usage is correctly typed. In fact, it found that `parse` was missing the `int` annotation on its first parameter.